src/index.html

```diff
@@ -26,54 +26,42 @@
+                <li> Delightful, flexible <span data-toggle="popover" data-content="These range from <a href='/tools'>general-purpose tools</a> to framework-specific ones like the <a href='https://flutter.io/getting-started'><code>flutter</code> tool</a>.">tooling</span>
+                </li>
+              </ul>
+              <p><a href="/guides/get-started" class="frontpage-slogan__get-started btn btn-default btn-lg">Get Started</a> </p>
             </h3>
-            <p><a href="/guides/get-started" class="frontpage-slogan__get-started btn btn-default btn-lg">Get Started</a> <a href="/install" class="btn btn-lg">Install Dart</a></p>
+            <p> <em>Click <span data-toggle="popover" data-content="Try clicking dotted underlines in the text above and in the code sample.">underlined text</span> to learn more.</em> </p>
         </div>
       </div>
     </div>
   </div>
 </div>
 </section>
 
-<section class="chip-grid-section">
+<section class="card-grid-section">
   <div class="container">
     <div class="row">
       <div class="col-sm-12">
         <div class="content">
-          <div class="chip-grid">
-            <a href="http://news.dartlang.org/" class="chip no-automatic-external">
-              <h3>News</h3>
-              <p>Follow the latest.</p>
-            </a>
-            <a href="https://api.dartlang.org/" class="chip no-automatic-external">
-              <h3>API</h3>
-              <p>Browse core libraries.</p>
-            </a>
-            <a href="https://pub.dartlang.org/" class="chip no-automatic-external">
-              <h3>Pub</h3>
-              <p>Find packages.</p>
-            </a>
-            <a href="{{site.webdev}}" class="chip no-automatic-external">
-              <h3>Dart webdev</h3>
-              <p>Build browser apps.</p>
+          <div class="card-grid">
+            <a href="/guides/language/language-tour" class="card no-automatic-external">
+              <h4>Language tour</h4>
+              <p>A walk through Dart's major language features.</p>
             </a>
-            <a href="https://flutter.io/" class="chip no-automatic-external">
-              <h3>Flutter</h3>
-              <p>Build mobile apps.</p>
+            <a href="/guides/libraries/library-tour" class="card no-automatic-external">
+              <h4>Library tour</h4>
+              <p>An example-driven introduction to Dart's core libraries.</p>
             </a>
-            <a href="/dart-vm" class="chip no-automatic-external">
-              <h3>Dart VM</h3>
-              <p>Build scripts, servers.</p>
+            <a href="https://pub.dartlang.org/" class="card no-automatic-external">
+              <h4>Dart packages</h4>
+              <p>An ecosystem of contributed libraries and tools.</p>
```

Comment:
Might consider using (some of) the text from the Pub front-page "Find and use packages to build Flutter and web apps with Dart." instead.

Reply:
That didn't turn out to fit well here, once I added the Platforms card.